src/main/java/uk/gov/hmcts/reform/sendletter/services/AuthService.java

```diff
@@ -1,49 +1,23 @@
 package uk.gov.hmcts.reform.sendletter.services;
 
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.authorisation.validators.AuthTokenValidator;
 import uk.gov.hmcts.reform.sendletter.exception.UnauthenticatedException;
-import uk.gov.hmcts.reform.sendletter.exception.UnauthorizedException;
-
-import java.util.Objects;
 
 @Component
 public class AuthService {
 
-    /** Name of the service that can update letter status. **/
-    private final String statusAccessorService;
     private final AuthTokenValidator authTokenValidator;
 
-    public AuthService(
-        @Value("${status-update-service-name}") String statusAccessorService,
```

Comment:
Do we need to remove this https://github.com/hmcts/send-letter-service/blob/master/src/integrationTest/resources/application.yaml#L1?

Reply:
yup!